ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -118,8 +131,9 @@ const styles = reactStyles({
     padding: '0 0.1rem',
   },
   pleaseComplete: {
```

Comment:
This matches the new mocks in *all cases* so this is a style change to the old page as well.

It's my personal style preference but you can convince me to change it back.